engine/runtime/src/main/scala/org/enso/compiler/context/ChangesetBuilder.scala

```diff
@@ -22,6 +30,7 @@ import scala.collection.mutable
 case class Changeset[A](
   source: A,
   ir: IR,
+  simpleUpdate: SimpleUpdate,
```

Comment:
missing doc. also unlike other params this can carry null? If it is optional then Scala's way would be to use Option

Reply:
Updated to `Option[SimpleUpdate]` in the latest changeset.